integration/isolated/update_buildpack_command_test.go

```diff
@@ -40,6 +40,38 @@ var _ = Describe("update-buildpack command", func() {
 		})
```

Comment:
Please add a test for updating a buildpack with a manifest that specifies a different stack. Per the original proposal, 
>  If the manifest.yml specifies a different stack, this is an error.
and there should be test ensuring that the error is bubbled up from the API.

Reply:
👍